arlas-core/src/main/java/io/arlas/server/app/ArlasServerConfiguration.java

```diff
@@ -70,6 +68,15 @@
     @JsonProperty("elastic-cluster")
     public String elasticcluster;
 
+    @JsonProperty("elastic-cloud-enable-ssl")
+    public Boolean elasticCloudEnableSsl;
+
+    @JsonProperty("elastic-cloud-credentials")
```

Comment:
`elastic-cloud` is not necessarily relevant : it could be a connection to a secured elasticsearch: `credentials` is enough.

Reply:
fixed